pytorch_lightning/trainer/trainer.py

```diff
@@ -1251,9 +1253,11 @@ def _pre_training_routine(self):
         # --------------------------
         # Pre-train
         # --------------------------
-        self.call_hook("on_pretrain_routine_start")
+        self._call_callback_hooks("on_pretrain_routine_start")
+        self._call_lightning_module_hook("on_pretrain_routine_start")
 
-        self.call_hook("on_pretrain_routine_end")
+        self._call_callback_hooks("on_pretrain_routine_end")
+        self._call_lightning_module_hook("on_pretrain_routine_end")
```

Comment:
quick ques: why `_call_callback_hooks` with an `s`?

Reply:
I see `callback hook` as a compound noun composed of two nouns, similar to `rain boot` or `car seat`. To make a compound noun plural you only pluralize the second noun, i.e. `rain boots`, `car seats`, `callback hooks`